src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -896,6 +895,15 @@ def validate_params(self):
                 manifest.is_folder_allowed()
             ), f"Invalid value for `folder` {manifest.folder} must be one of {manifest.get_allowed_folders()}"
 
+    def create_single_custom_manifest(self, filename: str = None, folder: str = None, base64_content: str = None):
+        return self.api_client.create_custom_manifest(self.id, filename, folder, base64_content)
+
+    def list_custom_manifests(self):
+        return self.api_client.list_custom_manifests(self.id)
+
+    def delete_custom_manifest(self, filename: str = None):
+        return self.api_client.delete_custom_manifest(self.id, filename)
```

Comment:
Are we sure that we need both this methods in `cluster.py` and `assisted_service_api.py`?

Reply:
this  is for us to be using the function from the cluster object , like other functions